Apps/WebClient/src/ClientApp/app/views/auth.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
Looks like this file is for testing only. We can keep it to make sure the authentication is working propertly.